action.yml

```diff
@@ -104,7 +108,7 @@ runs:
         mkdir -p "${SPOTBUGS_WORKING}"
         cd "${SPOTBUGS_WORKING}"
         SPOTBUGS_FILES=$(find "${GITHUB_WORKSPACE}/${INPUT_SPOTBUGS_TARGET}" -type f -name "${INPUT_SPOTBUGS_GLOB}" -exec echo -n {} \+)
-        "${SPOTBUGS_HOME}/bin/spotbugs" -textui -quiet -effort:max -low -bugCategories SECURITY -pluginList "${FINDSECBUGS_HOME}/findsecbugs-plugin-${INPUT_FINDSECBUGS_VERSION}.jar" -sarif=spotbugs.sarif ${SPOTBUGS_FILES}
+        "${SPOTBUGS_HOME}/bin/spotbugs" -maxHeap ${{ inputs.maxHeap }} -textui -quiet -effort:max -low -bugCategories SECURITY -pluginList "${FINDSECBUGS_HOME}/findsecbugs-plugin-${INPUT_FINDSECBUGS_VERSION}.jar" -sarif=spotbugs.sarif ${SPOTBUGS_FILES}
```

Comment:
Please can this go via an `INPUT_MAXHEAP` variable please? That allows proper escaping and avoids accidentally messing up the `spotbugs` command line.

Assign an env first:

```yaml
env:
  INPUT_MAXHEAP: ${{ inputs.maxHeap}}
```

and then use that with `"..."` around it:

```bash
"${SPOTBUGS_HOME}/bin/spotbugs" -maxHeap "${INPUTS_MAXHEAP}" -textui ...
```